pkg/common/cns-lib/volume/manager.go

```diff
@@ -180,10 +180,8 @@ func (m *defaultManager) ResetManager(ctx context.Context, vcenter *cnsvsphere.V
 	log := logger.GetLogger(ctx)
 	managerInstanceLock.Lock()
 	defer managerInstanceLock.Unlock()
-	if vcenter.Config.Host != managerInstance.virtualCenter.Config.Host {
-		log.Infof("Re-initializing defaultManager.virtualCenter")
-		managerInstance.virtualCenter = vcenter
-	}
+	log.Infof("Re-initializing defaultManager.virtualCenter")
+	managerInstance.virtualCenter = vcenter
 	m.virtualCenter.Config = vcenter.Config
```

Comment:
if `managerInstance.virtualCenter = vcenter` is executed then it has the latest config. I think we do not need line 185?

Reply:
Done